pydocx/openxml/markup_compatibility/fallback.py

```diff
@@ -6,10 +6,8 @@
 )
 
 from pydocx.models import XmlModel, XmlCollection
-from pydocx.openxml.wordprocessing.picture import Picture
 
 
 class Fallback(XmlModel):
     XML_TAG = 'Fallback'
-    # TODO #204: actually include all of the children defined in the spec.
-    children = XmlCollection(Picture)
+    children = XmlCollection(allow_all_children=True)
```

Comment:
Here's another idea: Make `children` a property. In the property, goes up two parent levels, and returns the children from that level. Would that work?